### ISSUE TITLE
GoToLine invalid status action is inaccessible

### ISSUE THREAD

**Issue by WebCoder49** [WebCoder49/code-input#159]
When the line number entered into the GoToLine box is invalid, it will turn red. Using colour only to signify this makes it invisible to screenreaders and could also be bad for some kinds of colourblind users. After this, pressing enter does nothing with no explanation.

A message should be shown to both screenreaders and the screen when it is invalid:
* ~~the easiest way to do this is to **replace the custom JavaScript validation of `input[type=text]` with HTML form validation of `input[type=number]`**, and use the `min` and `max` *as well as JavaScript validation methods as many minor browsers show range validation clearly* ([source](https://caniuse.com/input-number)).~~
* `input[type=text]` should be used, to support line:column navigation
